### PR TITLE
Use EU mirror instead of UK mirror

### PR DESCRIPTION
**Pull request by dgtlmoon**

### CODE REVIEW

**Comment by dgtlmoon** [comment]
Providing EU mirror which is way faster than mirrorservice (was getting 200kb/s here in CZ)

**Comment by Coornail** [comment]
I'm happy to change the mirror, but please don't remove my details from the maintainers and the readme.
(You are welcome to maintain a fork with these changes though.)

**Comment by dgtlmoon** [comment]
Ok hmm, i thought it was just the Dockerfile I had a pull request on, didnt realise the readme.md went across also :D sorry man!

**Comment by dgtlmoon** [comment]
ah, need to use a branch next time, got it

**Comment by dgtlmoon** [comment]
We are both listed

MAINTAINER  Leigh Morresi "dgtlmoon@gmail.com"
MAINTAINER  Kornel Lugosi "coornail@gmail.com"

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Docker image for ApacheSolr for Drupal search.
 
 To use:
 ```sh
-git clone git@github.com:Coornail/apachesolr-drupal-docker.git 
+git clone git@github.com:dgtlmoon/apachesolr-drupal-docker.git
```
Comment (Coornail) [inline README.md:9]: I'm not sure if you closed the issue on purpose, but let's keep this pointing to the current repo.

**Pull request closed without merging**